src/lib.rs

```diff
@@ -35,6 +36,15 @@ extern crate alloc;
 #[cfg(not(has_extern_crate_alloc))]
 extern crate std as alloc;
 
+#[cfg_attr(feature = "nightly", macro_use)]
+#[cfg(feature = "nightly")]
+#[cfg(doctest)]
+extern crate doc_comment;
```

Comment:
Isn't `macro_use` and `extern crate` mostly unnecessary now that we have 2018 edition? Can't we just do:
```rust
doc_comment::doctest!("../README.md");
```

Reply:
We absolutely can. If you only target 2018 edition, then I'll remove those lines.

Reply:
Yes this crate only targets 2018 edition, please remove those lines.